.github/workflows/testing.yml

```diff
@@ -63,3 +65,58 @@ jobs:
         run: |
           cd app/
           python manage.py check
+  lighthouse:
+    runs-on: ubuntu-latest # operating system your code will run on
+    services:
+      postgres:
+        image: postgres:16
+        env:
+          POSTGRES_USER: sadilar
+          POSTGRES_PASSWORD: sadilar
+          POSTGRES_DB: term_db
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-test.txt
+          sudo apt-get install -y gettext
+          npm install -g @lhci/cli@0.14.x
+      - name: Setup LHCI files (env & entrypoint)
+        run: chmod +x entrypoint.sh && cp .env.lhci app/.env
+      - name: Run Django fixtures
+        run: |
+          cd app
+          python manage.py migrate --no-input
+          python manage.py loaddata fixtures/institution.json
+          python manage.py loaddata fixtures/language.json
+          python manage.py loaddata fixtures/projects.json
+          python manage.py loaddata fixtures/subjects.json
+          python manage.py import_documents general/tests/files/
+      - name: Run Lighthouse
+        run: cd app && lhci autorun
+        env:
+          DJANGO_SETTINGS_MODULE: app.settings
+          DATABASE_URL: postgres://sadilar:sadilar@localhost:5432/term_db
```

Comment:
The DB settings are now in this variable and in the .env file. I would prefer no duplication (but I see you had a bad example to work from). I'm not familiar with if/how this variable works (it is supported by django-environ), so easiest (for me) might be to leave it out here. If we need the .env file anyway for other settings, I guess leaving it out is cleanest.

Reply:
Looks like `DJANGO_SETTINGS_MODULE` is not set in the file; I could add it to .env.lhci?